ui/cypress/integration/integration/configuration_spec.js

```diff
@@ -0,0 +1,64 @@
+describe('Configuration Management', () => {
+  beforeEach(() => {
+    // Prepare Stub Interception for Initial Filters
+    cy.intercept('GET', '/api/filter**', { fixture : 'configuration/filter-stubs/initial-filters.json' }).as('getInitialFilters');
```

Comment:
```suggestion
    cy.intercept('GET', '/api/filter**', { fixture : 'configuration/filter-stubs/initial-filters.json' }).as('getInitialFilters');
    cy.selectProviderNone();
```
Without selecting providers, we won't be able to perform operations, I think.

Reply:
@Abhishek-kumar09 thanks for suggestion, although in a prior commit adding the `cy.selectProviderNone()` custom command threw errors:
https://github.com/meshery/meshery/pull/4717/commits/288775510ef8dcd3cc6a5e0ebc822b34edf73894

Reply:
@MarioArriaga92 I think, I can't push some changes to your branch. How can I do that? Any idea.

Reply:
> @Abhishek-kumar09 thanks for suggestion, although in a prior commit adding the `cy.selectProviderNone()` custom command threw errors: [2887755](https://github.com/meshery/meshery/commit/288775510ef8dcd3cc6a5e0ebc822b34edf73894)

Oh I see, I just ran all the tests successfully after this. Maybe you can re-assure.

Reply:
@Abhishek-kumar09 let's keep in mind that tests inside `/ui/cypress/integration/integration/` folder run in .github/workflows/ci.yml file's [`tests-ui` job](https://github.com/meshery/meshery/blob/master/.github/workflows/ci.yml#L108) which only relies on `build-ui` job (`Run tests` run step only starts Meshery UI in current Github Actions runner during [`npm run ci-test-integration`](https://github.com/meshery/meshery/blob/master/.github/workflows/ci.yml#L136)).

Here's failure that occured when suggested custom command was used in `beforeEach` hook:
https://github.com/meshery/meshery/runs/4412890391?check_suite_focus=true

Reply:
Current `Meshery / UI integration tests (pull_request)` check in PR footer shows success on latest run and drilling into its 'Details' would bring up run output:
![Screen Shot 2021-12-09 at 9 43 05](https://user-images.githubusercontent.com/23443575/145427938-5b991cff-6722-4252-bc89-f348cfffeaa3.png)

Reply:
@Abhishek-kumar09 Tests run successfully with `npm run ci-test-integration` right from /ui/ folder so resolving this conversation, but let's keep conversation going about simplifying working with cypress locally right on slack 🔥👌🙏🏼

Reply:
Sure :muscle: